pages/mapping.js

```diff
@@ -12,10 +12,11 @@ export default function mapping() {
 
 	const [answerCount, setAnswerCount] = useState(0);
 	const [counter, setCounter] = useState(0);
-	const [question, setQuestion] = useState({ id: 0, lat: 0, lon: 0, answer: ''});
+	const [question, setQuestion] = useState({ id: 0, lat: 0, lon: 0, country_name: '', country_code: '', answer: ''});
```

Comment:
In the state, you only need to keep one of the two, that would be `country_code`. Make the case otherwise.